services/node_manager/pom.xml

```diff
@@ -109,6 +109,26 @@
 					</systemPropertyVariables>
```

Comment:
Indent in node_manager/pom.xml: use "empty space" instead of "tab"

Reply:
I replaced "tab" with empty space as much as I can. But if it needs more, let me know.